### PR TITLE
CB-4041 extend cluster's security rules to nginx with control plane cidrs

### PR DESCRIPTION
**Pull request by schfeca75**
@keyki pls review

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackV4RequestToStackConverter.java [hunk 1]
```diff
@@ -152,9 +163,35 @@ public Stack convert(StackV4Request source) {
...
+    private void extendGatewaySecurityGroupWithDefaultGatewayCidrs(Stack stack) {
+        Set<InstanceGroup> gateways =
+                stack.getInstanceGroups().stream().filter(ig -> InstanceGroupType.isGateway(ig.getInstanceGroupType())).collect(Collectors.toSet());
+        Set<String> defaultGatewayCidrs = defaultGatewayCidr
+                .stream()
+                .filter(StringUtils::isNotBlank)
+                .collect(Collectors.toSet());
+        for (InstanceGroup gateway : gateways) {
+            Set<SecurityRule> rules = gateway.getSecurityGroup().getSecurityRules();
+            if (!defaultGatewayCidrs.isEmpty()) {
```
Comment (lajosrodek) [inline core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackV4RequestToStackConverter.java]: I suggest pulling the `if` outside the `for` loop as the condition is constant throughout all iterations.

File: core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackV4RequestToStackConverter.java [hunk 2]
```diff
@@ -107,6 +112,12 @@
     @Value("${cb.platform.default.regions:}")
     private String defaultRegions;
 
+    @Value("#{'${cb.default.gateway.cidr:}'.split(',')}")
+    private Set<String> defaultGatewayCidr;
+
+    @Value("${cb.nginx.port:9443}")
```
Comment (keyki) [inline core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackV4RequestToStackConverter.java]: We shouldn't have default value here as it's part of the app.yml

File: cloud-common/src/main/resources/application.yml [hunk 3]
```diff
@@ -83,6 +83,7 @@ cb:
     AZURE: 50
     GCP: 50
   enabled.linux.types: redhat6,redhat7,centos6,centos7
+  default.gateway.cidr:
```
Comment (keyki) [inline cloud-common/src/main/resources/application.yml]: This should have a value for 0.0.0.0/0

File: freeipa/src/main/resources/application.yml [hunk 4]
```diff
@@ -24,6 +24,7 @@ management:
 freeipa:
   cert.dir: /certs/
   client.id: freeipa
+  default.gateway.cidr:
```
Comment (keyki) [inline freeipa/src/main/resources/application.yml]: This should have a value for 0.0.0.0/0

**Comment by keyki** [comment]
In FreeIPA application.yml there is a part
```
  security.default:
    ports: 22,443,9443
    cidr: 0.0.0.0/0
```
I think this should be removed.

**Comment by schfeca75** [comment]
> In FreeIPA application.yml there is a part
> 
> ```
>   security.default:
>     ports: 22,443,9443
>     cidr: 0.0.0.0/0
> ```
> 
> I think this should be removed.

Done

**Comment by schfeca75** [comment]
> No loglines were added

Logs are added

**Pull request merged**